README.adoc

```diff
@@ -9,14 +9,17 @@ $ java -jar ./bandcamp-collection-downloader.jar -d myMusicFolder myBandcampAcco
 image::img/example-output.gif[]
 
 ....
-Usage: <main class> [-eh] [-c=<pathToCookiesFile>] [-d=<pathToDownloadFolder>] [-f=<audioFormat>] [-j=<jobs>] [-r=<retries>]
+Usage: <main class> [-ehv] [-c=<pathToCookiesFile>] [-d=<pathToDownloadFolder>] [-f=<audioFormat>] [-j=<jobs>] [-r=<retries>]
                     [-t=<timeout>] <bandcampUser>
       <bandcampUser>        The bandcamp user account from which all releases must be downloaded.
   -c, --cookies-file=<pathToCookiesFile>
-                            A JSON file with valid bandcamp credential cookies.
-                            "Cookie Quick Manager" can be used to obtain this file after logging into bandcamp.
-                            (visit https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/).
-                            If no cookies file is provided, cookies from the local Firefox installation are used (Windows and Linux
+                            A JSON or txt file with valid bandcamp credential cookies.
+                            "Cookie Quick Manager" can be used to obtain a JSON file after logging into bandcamp.
+                            (visit https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/)
+                            "or the "cookies.txt" extension"
+                            (https://chrome.google.com/webstore/detail/cookiestxt/njabckikapfpffapmjgojcnbfjonfjfg?hl=en)
+                            can be used to obtain a cookies.txt file on Chrome.
```

Comment:
Here I would rephrase like this:

A file containing valid bandcamp credential cookies.
The file must either be obtained using the Firefox extension "Cookie Quick Manager" (https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/) or using the Chrome extension "cookies.txt" (https://chrome.google.com/webstore/detail/cookiestxt/njabckikapfpffapmjgojcnbfjonfjfg).
If no cookies file is provided, cookies from the local Firefox installation are used (Windows and Linux).

WDYT?

Reply:
I think that is fine, I replaced it with this text